data/pnmtopng.cmake

```diff
@@ -0,0 +1,13 @@
+# requires list of files set using -Din_file="file1" -Dout_file="file2"
+
+execute_process (
+    COMMAND
+        pnmtopng
+    INPUT_FILE
+        ${in_file}
+    OUTPUT_FILE
+        ${out_file}
+    OUTPUT_VARIABLE
+        output
+    ERROR_VARIABLE
+        error)
```

Comment:
This is a copy from
https://github.com/ev3dev/ev3devKit/blob/ev3dev-jessie/data/pnmtopng.cmake
which neither has a newline at the end of the file. I've sent you a PR for that to keep it consistent.

Done.